### PR TITLE
[hevce][refactored] Enabled HEVCe refactored code path for Linux SKL

### PR DESCRIPTION
**Pull request by alariono**
Signed-off-by: Andrey Larionov <andrey.larionov@intel.com>

### CODE REVIEW

File: _studio/shared/include/mfx_config.h [hunk 1]
```diff
@@ -123,6 +123,15 @@
     #endif
 #endif
 
+#if !defined(ANDROID)
+#define MFX_ENABLE_HEVCEHW_REFACTORING
```
Comment (dvrogozh) [inline _studio/shared/include/mfx_config.h]: Why do we diverge for Android?

File: _studio/shared/include/mfx_config.h [hunk 2]
```diff
@@ -123,6 +123,15 @@
...
+#endif //!defined(ANDROID)
+
+#if defined(MFX_ENABLE_HEVCEHW_REFACTORING)
+#define MFX_ENABLE_HEVCEHW_REFACTORING_WIN_SKL
```
Comment (dvrogozh) [inline _studio/shared/include/mfx_config.h]: This will enable SKL only or it will enable gen9+ support?

Comment (mgonchar) [reply _studio/shared/include/mfx_config.h]: Only gen9 graphics, not only SKL

Comment (dvrogozh) [reply _studio/shared/include/mfx_config.h]: In this case this macro name is wrong and misleading. Please, rename to MFX_ENABLE_HEVCEHW_REFACTORING_WIN_GEN9 and MFX_ENABLE_HEVCEHW_REFACTORING_LIN_GEN9 first.

Secondly, why we switch new code for GEN9 only?

Comment (mgonchar) [reply _studio/shared/include/mfx_config.h]: it is agreement with HEVCe scrum team to make switch platform by platform. now SKL has acceptable passrates and all perf, stress, quality data so it is ready for switch

File: _studio/shared/include/mfx_config.h (hunk 1 above)
Comment (mgonchar) [reply _studio/shared/include/mfx_config.h]: it is related to some build issues. @eistarov could you comment here?

Comment (eistarov) [reply _studio/shared/include/mfx_config.h]: Need to update Android.mk and fix couple of warnings - see PR draft #1858

**Comment by alariono** [comment]
ES done it in PR https://github.com/Intel-Media-SDK/MediaSDK/pull/1858

**Comment by dvrogozh** [comment]
what is ES?

#1858 looks better. Thank you.

File: _studio/shared/include/mfx_config.h (hunk 2 above)
Comment (dvrogozh) [reply _studio/shared/include/mfx_config.h]: ok

**Pull request closed without merging**